### PR TITLE
Add links to Slack

### LINKED ISSUE #1322: Setup a Slack team for maintainers and contributors
From the [ROADMAP](https://github.com/github/hubot/pull/1321)

> - [ ] Choose a chat platform for maintainers and contributors, and post notices in various existing places (#hubot on freenode, github/hubot on Gitter). Slack is the obvious choice here.

- [x] create a Hubot slack team
- [x] [Create and configure a hubot instance](https://github.com/hubot-scripts/hubot-for-hubot)
- [x] Setup IRC gateway to connect Slack and #hubot on Freenode.
- [x] Setup [slackin](https://github.com/rauchg/slackin)
- [x] Create default channels
  - [x] #help - for users getting help with the project
  - [x] #contributors - for discussions amongst maintainers & contributors
  - [x] #maintainers - rarely used private channel for confidential discussions
  - [x] #random - for random discussions about bots, technology, and sandwiches
- [x] ~~Wait until Code of Conduct has been added to the project~~
- [x] Add link to join Slack to the README
- [x] Post notice in #hubot on IRC
- [x] Post notice in Gitter

### PR DESCRIPTION
**Pull request by bkeepers**
This adds links to our shiny new Slack account to the README.

Closes #1322

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ for details on getting up and running with your very own robot friend.
 In most cases, you'll probably never have to hack on this repo directly if you
 are building your own bot. But if you do, check out [CONTRIBUTING.md](CONTRIBUTING.md)
 
-If you'd like to chat with Hubot users and developers, drop by #hubot on FreeNode IRC.
+If you'd like to chat with Hubot users and developers, [join us on Slack](https://hubot-slackin.herokuapp.com/).
```
Comment (technicalpickles) [inline README.md]: Should we mention that #hubot still exists, but that Slack is the official place?

Comment (bkeepers) [inline README.md]: Slack has official IRC support. Are ther advantages to using Freenode over that?

My preference would be to reduce the surface area here and use one service.

Comment (mose) [inline README.md]: I agree, it's simpler with just slack. Still we would need to regain control on the irc channel to setup a link to slack on the `/topic`

Comment (michaelansel) [inline README.md]: @technicalpickles has control of the channel through ChanServ

**Pull request merged**